### ISSUE TITLE
Target class [router] does not exist

### ISSUE THREAD

**Issue by kb6673** [laravel/pint#19]
- Pint Version: 0.1.5
- PHP Version: 8.0.20

### Description:
I've tried this in two local Lara projects.  Running `./vendor/bin/pint` results in an error shown below

![image](https://user-images.githubusercontent.com/1001484/175273058-edc27503-9d93-4411-ba12-31ff73a5dab3.png)

### Steps To Reproduce:

`composer require laravel/pint --dev`

then

`./vendor/bin/pint`

Am I missing something?

**Comment by driesvints** [comment]
I don't get this issue. What Laravel version are you using?

**Comment by kb6673** [comment]
First attempt was 8.83.16. Second attempt was 9.17.0

**Comment by driesvints** [comment]
Heya, thanks for reporting.

We'll need more info and/or code to debug this further. Can you please create a repository with the command below, commit the code that reproduces the issue as separate commits on the main/master branch and share the repository here? Please make sure that you have the [latest version of the Laravel installer](https://github.com/laravel/installer) in order to run this command. Please also make sure you have both Git & [the GitHub CLI tool](https://cli.github.com) properly set up.

```bash
laravel new bug-report --github="--public"
```

Please do not amend and create a separate commit with your custom changes. After you've posted the repository, we'll try to reproduce the issue.

Thanks!

**Comment by nunomaduro** [comment]
Can you share your global "composer.lock" please?

**Comment by nunomaduro** [comment]
We will re-open the issue once you share your "composer.lock" file.

**Comment by deniscsz** [comment]
@nunomaduro I just faced the same issue. 

My composer.lock: https://gist.github.com/deniscsz/5c2a05a4046e648643bcd403f10295de

The output of `php artisan about`:
<img width="922" alt="Screen Shot 2022-08-14 at 23 48 12" src="https://user-images.githubusercontent.com/2111143/184569059-cb5e6d94-ecdd-435e-adf3-7f87f10fc12c.png">

**Comment by deniscsz** [comment]
I didn't know why but after recreate my docker's containers the error is gone. I believe it wasn't an issue of Pint.

**Comment by flxsource** [comment]
@kb6673 @deniscsz 

Did any of you recently install the New Relic PHP agent? Does removing the New Relic config from the CLI conf allow Pint to run?

**Comment by deniscsz** [comment]
@flxsource Yes! When I removed the new relic php agent, the problem was gone.

**Comment by kb6673** [comment]
> Did any of you recently install the New Relic PHP agent?
Not recently - it's always bee there :)

I'll disable it for the CLI and confirm @deniscsz's findings

**Comment by rohan-mistry** [comment]
@flxsource @kb6673 @deniscsz Do you know why disabling new relic solved the pint error ? I am trying to install pint in our core php project and we have new relic already installed. We cannot disable new relic for our system. If you could provide some insights that would be very helpful
